README.adoc

```diff
@@ -88,7 +88,7 @@ Your images will also need network connectivity to be able to reach an actual OT
 * `INITRAMFS_IMAGE` - initramfs/initrd image that is used as a proxy while booting into OSTree deployment. Do not change this setting unless you are sure that your initramfs can serve as such a proxy.
 * `SOTA_PACKED_CREDENTIALS` - when set, your ostree commit will be pushed to a remote repo as a bitbake step. This should be the path to a zipped credentials file in https://github.com/advancedtelematic/aktualizr/blob/master/docs/credentials.adoc[the format accepted by garage-push].
 * `SOTA_DEPLOY_CREDENTIALS` - when set to '1' (default value), deploys credentials to the built image. Override it in `local.conf` to built a generic image that can be provisioned manually after the build.
-* `SOTA_CLIENT_PROV` - which provisioning method to use. Valid options are https://github.com/advancedtelematic/aktualizr/blob/master/docs/automatic-provisioning.adoc[`aktualizr-auto-prov`], https://github.com/advancedtelematic/aktualizr/blob/master/docs/implicit-provisioning.adoc[`aktualizr-ca-implicit-prov`], and https://github.com/advancedtelematic/aktualizr/blob/master/docs/hsm-provisioning.adoc[`aktualizr-hsm-prov`]. The default is `aktualizr-auto-prov`. This can also be set to an empty string to avoid using a provisioning recipe.
+* `SOTA_CLIENT_PROV` - which provisioning method to use. Valid options are [`aktualizr-auto-prov`], [`aktualizr-ca-implicit-prov`], and [`aktualizr-hsm-prov`]. For more information on these provisioning methods, see the https://docs.ota.here.com/client-config/client-provisioning-methods.html[OTA Connect documentation]. The default is `aktualizr-auto-prov`. This can also be set to an empty string to avoid using a provisioning recipe.
```

Comment:
What's the point of keeping the brackets if there is no link anymore? I think it makes more sense to fix the things in aktualizr first and then come back and fix the links here.

There are also a lot of code changes to take care of as part of this rename. Do you want me to work on that, and would you prefer to keep that as a separate branch/PR?

Reply:
@patrickvacek, could we get rid of '_auto_' and '_implicit_' word for the configuration options ?

Reply:
Yes, that is the plan. However, it isn't quite as easy as just changing it, we have to update tests and make sure everything still works. That first requires updating aktualizr and then coming back to this here.

Reply:
Sorry the brackets were an oversight. Fixed now. Other issues to be handled in separate PR: